src/quadpype/hosts/tvpaint/plugins/publish/extract_sequence.py

```diff
@@ -428,7 +429,7 @@ def render_review(
         return output_filepaths_by_frame_index, thumbnail_filepath
 
     def render(
-        self, output_dir, mark_in, mark_out, layers, ignore_layers_transparency, apply_background, resolution, export_frames
+        self, output_dir, mark_in, mark_out, layers, ignore_layers_transparency, apply_bg_back_command, resolution, export_frames
```

Comment:
Should be simply a bool, not a george script command since we won't use it in this method